samples/samples/connection_sample.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+
+from google.cloud import spanner
+from google.cloud import spanner_dbapi
+
+from google.cloud.spanner_dbapi import connection, InterfaceError
+
+
+def list_all():
+    spanner_client = spanner.Client()
+
+    for inst in spanner_client.list_instances():
+        print(inst.name)
+        for db in inst.list_databases():
+            print(db.name)
+
+
+def create_connection(instance_id, database_id):
+    my_connection = spanner_dbapi.connect(instance_id, database_id)
+
+    print("Connection is created.")
```

Comment:
We can probably union connection and cursor creation samples into one sample. They are small and will never be used in separate.